components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  pathname,
+  locale,
+}) {
+  const urlPrefix = fallbackLocale === locale ? '' : `${locale}.`
+  const image = `https://${urlPrefix}glamorous.rocks/static/images/banner.png`
```

Comment:
`getInitialLocaleProps` is being used as function for `getInitialProps` in `withContent` function.

`getInitialLocaleProps` is currently being used together with `withContent` for component(s) in `pages` and I tried tweaking it in my local to return `host`. Works fine on the server to return `host`.

Also, I certainly agree with you @paulmolluzzo on breaking this into separate issue. 👍